frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.module.scss

```diff
@@ -331,42 +331,14 @@
     padding: 10px 0 10px 0;
   }
 
-  .EmailAddressForm {
```

Comment:
I think we still want this and the text field to have a border radius and outline per the mocks

Reply:
I was on the fence about this change but my preference would be to migrate these components to the material standard like the rest of the site even if it means diverging from the mocks. I was thinking we might reach out to UX to see if we could get input on adapting the current design so we can be consistent with the material standard.

I think by reusing react material components rather than styling our own we get a lot of professional looking bonuses for free (like the hovering and click effects) and also reduce the chances of custom-built styles going awry unexpectedly in different browsers. For instance the current sign up looks kind of odd to me because my browser autofills it:

![Screen Shot 2021-04-08 at 9 16 28 PM](https://user-images.githubusercontent.com/6402808/114128918-b69ccb00-98ba-11eb-84be-e0e123fc2e9b.png)

Reply:
That makes total sense & I agree -- lemme ping Chi for her input on the new look and then reply to this if she suggests any changes?

Reply:
That sounds great- thank you Elizabeth :D